### PR TITLE
Test landyachting bot

### PR DESCRIPTION
**Pull request by splendidcomputer**
Branching works.

### CODE REVIEW

File: apps/chat-ui/refactorAgent.js [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+
+const outputChain = new RunnableLambda({
+  func: (input) => {
+    return {
+      answer: input.answerFromPrompt.answer,
+    }
+  },
+})
+
+// const debuggerChain = (tag) =>
```
Comment (splendidcomputer) [inline apps/chat-ui/refactorAgent.js:136]: I kept it becuase, we might need it later.

File: apps/chat-ui/refactorAgent.js
Comment (inpaceee) [inline apps/chat-ui/refactorAgent.js:1]: it seems like this file is redundant now that there is refactorAgent2.js? If that's the case, don't forget to rename it to just agent.js and delete commented out functions/variables

File: apps/chat-ui/package.json [hunk 2]
```diff
@@ -29,6 +30,7 @@
     "mammoth": "^1.8.0",
     "nodemon": "^3.0.2",
     "openai": "^4.52.7",
+    "pdf-parse": "^1.1.1",
```
Comment (inpaceee) [inline apps/chat-ui/package.json:33]: I think this is not used?
```suggestion
```

File: apps/chat-ui/refactorAgent2.js [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+        outputChain,
+      ]),
+    ),
+  )
+
+  console.log(
+    await mainChain.invoke('Was muss ich in Greifswald unbedingt ansehen?'),
+    // await mainChain.invoke('Was muss ich im Verzasca Tal unbedingt ansehen?'),
+  )
+}
```
Comment (inpaceee) [inline apps/chat-ui/refactorAgent2.js:145]:
```suggestion

```

**Pull request closed without merging**